### PR TITLE
Add new tutorials to doc changelog

### PR DESCRIPTION
**Pull request by merumelu**
Add visual shaders and microphone tutorials and update the version numbers in the first paragraph.

### CODE REVIEW

**Comment by clayjohn** [comment]
If we are saying that this lists the new tutorials since 3.1 we need to remove all the tutorials that were added before 3.1 was released.

**Comment by merumelu** [comment]
Should we reword it to be version independet instead? Something like this.

> Godot's documentation is continually being improved. With each engine release
> come many new tutorials, fixes and updates for old tutorials, and many updates
> to the :ref:`class reference <toc-class-ref>`. Below is an overview of new tutorials
> and the version they were added.

**Comment by akien-mga** [comment]
It's already version-dependent, the intro paragraph was just wrong. If you scroll down there's the changelog for the 3.1 updates below the latest updates.

**Comment by mhilbrunner** [comment]
Wow, this has been sitting here for a while. :) Sorry!

I'll merge this now, and we should cherrypick this to the 3.2 branch - I think making the initial paragraph more version independent is still a good idea, even if we need to keep the other parts of the document updated manually.

**Comment by mhilbrunner** [comment]
Thank you!
